packages/office-ui-fabric-react/src/components/Toggle/Toggle.tsx

```diff
@@ -14,7 +14,7 @@ import { Label } from '../../Label';
 import {
   customizable
 } from '../../Utilities';
-import { getClassNames } from './Toggle.classNames';
+import { getClassNames } from './Toggle.styles';
```

Comment:
Looking at the master branch, I think this is supposed to be classNames. Have you pulled from master recently?

Reply:
@battletoilet said that was probably an old name, so I changed it.

Reply:
The others all use "styles".

Reply:
Can you verify this works? This doesn't look right.